platform/pulpcore/app/serializers/user.py

```diff
@@ -85,15 +79,7 @@ def __init__(self, *args, **kwargs):
         if not settings.JWT_AUTH.get("JWT_ALLOW_SETTING_USER_SECRET"):
```

Comment:
This line is causing jwt_secret to not be displayed even when DEBUG mode is True. I would suggest moving this check to the jwt_secret serialzier field as:
`read_only=settings.JWT_AUTH.get("JWT_ALLOW_SETTING_USER_SECRET") `

Reply:
After talking to @daviddavis I will file a new issue for this since we are not sure what the correct behavior is supposed to be.